CHANGELOG.md

```diff
@@ -236,7 +237,7 @@ poshRuntime.stopOfferService(myServiceDescription);
 poshRuntime.findService({"ServiceA", iox::capro::AnyInstanceString});
 
 // after
-serviceDiscovery.findService("ServiceA", Wildcard);
+serviceDiscovery.findService("ServiceA", Wildcard, Wildcard);
```

Comment:
Since the include has also changed could you please add the new include directive needed to use the service discovery - only if it is required.